Assets/GothicVR/Scenes/Bootstrap.unity

```diff
@@ -1716,6 +1716,7 @@ MonoBehaviour:
   arrowmaterial: {fileID: 2100000, guid: dc6ce4e18c065d34f9037b5b60259872, type: 2}
   fillermaterial: {fileID: 2100000, guid: 7f15366450c487344a1e66a13b22a847, type: 2}
   skymaterial: {fileID: 2100000, guid: 39ac659c922b5b1489714aacd9b952e9, type: 2}
+  mapmaterial: {fileID: 2100000, guid: 6edba26a8e8b5c3488fd2eb615350942, type: 2}
```

Comment:
What is MapMaterial? Why do we have this file stored in the game, not via dynamic loading?

Reply:
It's similar to sky or button. I use an empty material, that is assigned the correct texture, when we have loaded the original game files.